public/main.js

```diff
@@ -0,0 +1,32 @@
+function fetchPOST(url, data, callback) {
```

Comment:
where did all this code come from ??

Reply:
This is our fetch functions for querying the server from the front end. There is one for GET method and 1 for post.